apps/showcase/src/assets/trainings/sdk/steps/angular-integration/instructions.md

```diff
@@ -0,0 +1,40 @@
+When dealing with an Angular project, you need to ensure that your `ApiClient` will be shared accross
+your application. The Otter framework provides the `ApiManager` service to manage your API collection.
+
+### Objective
+- Leverage the `ApiManager` service to access two different clients to retrieve the list of available pets and submit an order for the first pet returned.
+- Add a plugin to the `OrderApi` to log each time a call is sent.
+
+### Prerequisite
+- The package `@o3r/apis-manager` needs to be installed in the project with `npm install @o3r/apis-manager` (which has already been done for you).
```

Comment:
Should we do it for them or ask them to do it in the terminal ?

Reply:
It is done already, we just inform them that this is part of the @o3r/api-managers, but maybe this is not clear here